src/main/scala/scala2021/ashinkarev/task08/Frame.scala

```diff
@@ -0,0 +1,25 @@
+package scala2021.ashinkarev.task08
+
+
+case class Frame(
+  number: Int,
+  throws: List[Int] = List(),
+) {
+  def score = throws.sum;
+
+  def isStrike = if (throws.isEmpty) false else throws(0) == 10;
+  def isSpare = if (throws.length < 2) false else throws(0) + throws(1) == 10;
```

Comment:
Same as previous comment. 
Also, the best approach is prevent creating empty lists while parsing(probably create some domain object or add validation while parsing - both cases are ok)